core/tests/unittests/bayesopt/gpmxnet/test_comparison_gpy.py

```diff
@@ -0,0 +1,316 @@
+# import os
```

Comment:
Should we remove this file?

Reply:
You can remove it, but I will then add it back in once the modularization is done. #695

Reply:
Removed